pom.xml

```diff
@@ -184,6 +184,7 @@
         <module>legend-engine-testable</module>
         <module>legend-engine-executionPlan-execution-authorizer</module>
         <module>legend-engine-xt-relationalStore-executionPlan-authorizer</module>
+        <module>legend-engine-xt-relationalStore-bigquery-execution-tests</module>
```

Comment:
add this line in the place where all the other big query modules are listed

Reply:
done